zerver/views/events_register.py

```diff
@@ -22,17 +22,22 @@ def _default_narrow(user_profile: UserProfile,
         narrow = [['stream', default_stream.name]]
     return narrow
 
+EventTypesT = Iterable[str]
+NarrowT = Iterable[Sequence[Text]]
 @has_request_variables
-def events_register_backend(request, user_profile,
-                            apply_markdown=REQ(default=False, validator=check_bool),
-                            client_gravatar=REQ(default=False, validator=check_bool),
-                            all_public_streams=REQ(default=None, validator=check_bool),
-                            include_subscribers=REQ(default=False, validator=check_bool),
-                            event_types=REQ(validator=check_list(check_string), default=None),
-                            fetch_event_types=REQ(validator=check_list(check_string), default=None),
-                            narrow=REQ(validator=check_list(check_list(check_string, length=2)), default=[]),
-                            queue_lifespan_secs=REQ(converter=int, default=0)):
-    # type: (HttpRequest, UserProfile, bool, bool, Optional[bool], bool, Optional[Iterable[str]], Optional[Iterable[str]], Iterable[Sequence[Text]], int) -> HttpResponse
+def events_register_backend(request: HttpRequest, user_profile: UserProfile,
+                            apply_markdown: bool=REQ(default=False, validator=check_bool),
+                            client_gravatar: bool=REQ(default=False, validator=check_bool),
+                            all_public_streams: Optional[bool]=REQ(default=None, validator=check_bool),
+                            include_subscribers: bool=REQ(default=False, validator=check_bool),
+                            event_types: Optional[EventTypesT]=REQ(validator=check_list(check_string),
+                                                                   default=None),
+                            fetch_event_types: Optional[EventTypesT]=REQ(validator=check_list(check_string),
+                                                                         default=None),
+                            narrow: NarrowT=REQ(validator=check_list(check_list(check_string, length=2)),
+                                                default=[]),
+                            queue_lifespan_secs: int=REQ(converter=int, default=0)
+                            ) -> HttpResponse:
```

Comment:
You could start the arguments on a new line, and indent it at 8 spaces.
Something like: https://github.com/zulip/zulip/pull/7674/files#diff-237f2d7c30a5fdd38c7f1e2d8f6734eeR136

Reply:
Didn't know about this!

Reply:
Hmmm, the linter still says "continuation under indented for visual indent [E128]" for 8 spaces.

Reply:
I see, I have to do it starting from the first argument.